<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DbfToMySqlDumpDialog</class>
 <widget class="QDialog" name="DbfToMySqlDumpDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Формирование файла дампов MySql из DBF</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="7" column="0" colspan="4">
    <widget class="QGroupBox" name="gbOutStmtText">
     <property name="title">
      <string>Текст полученного mysql dump файла</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QPlainTextEdit" name="outStmtText"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="btnGenerate">
     <property name="text">
      <string>Сгенерировать</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="btnClose">
     <property name="text">
      <string>Закрыть</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="7">
    <widget class="QGroupBox" name="gbSourceFilesList">
     <property name="title">
      <string>Исходные файлы</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QListWidget" name="lstSourceFiles">
        <property name="selectionMode">
         <enum>QAbstractItemView::MultiSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="btnAddFiles">
        <property name="text">
         <string>Добавить файл(ы) ...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="btnDelFiles">
        <property name="text">
         <string>Удалить</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="QCheckBox" name="chkSaveToFile">
     <property name="text">
      <string>Сохранить результат в файл</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLabel" name="lblMySqlCodec">
     <property name="text">
      <string>MySql Charset:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="3">
    <widget class="QComboBox" name="cmbMySqlCodec">
     <item>
      <property name="text">
       <string>utf8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>cp1251</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>cp866</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QLabel" name="lblDbfCodec">
     <property name="text">
      <string>Кодировка DBF:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="3">
    <widget class="QComboBox" name="cmbDbfCodec">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <item>
      <property name="text">
       <string>utf8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>cp1251</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>cp866</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>DbfToMySqlDumpDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>526</x>
     <y>470</y>
    </hint>
    <hint type="destinationlabel">
     <x>413</x>
     <y>460</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
